config/config.py.template

```diff
@@ -429,6 +429,26 @@ MLLIB_GLM_REGRESSION_TEST_OPTS = MLLIB_GLM_TEST_OPTS + [
 MLLIB_TESTS += [("glm-regression", MLLIB_PERF_TEST_RUNNER, SCALE_FACTOR,
     MLLIB_JAVA_OPTS, [ConstantOption("glm-regression")] + MLLIB_GLM_REGRESSION_TEST_OPTS)]
 
+if MLLIB_SPARK_VERSION >= 1.5:
+    MLLIB_GLM_ELASTIC_NET_REGRESSION_TEST_OPTS = MLLIB_REGRESSION_CLASSIFICATION_TEST_OPTS + [
```

Comment:
I am getting

```
Missing required option(s) ['step-size', 'intercept', 'epsilon']
```

when I run this, so I think you will need to add those options here